boot/freeldr/bootsect/pc98/fat12fdd.S

```diff
@@ -357,33 +360,14 @@ PutChar:
     push di
     push es
 
-    push VRAM_TEXTMODE_ADDRESS
+    push word ptr BP_REL(VramSegment)
     pop es
-    mov di, word ptr BP_REL(VRAMOffset)         // ES:DI = 0A000:VRAMOffset
-
-    cmp di, VGA_WIDTH * VGA_HEIGHT * 2          // Needs to scroll
-    jb short .PutCharWrite
-
```

Comment:
These checks are not needed anymore as well?

Reply:
Yes, it's overkill at this stage.